go/pkg/filemetadata/filemetadata.go

```diff
@@ -79,6 +100,18 @@ func Compute(filename string) *Metadata {
 		md.IsDirectory = true
 		return md
 	}
+
+	if len(XattrName) > 0 {
```

Comment:
Should we throw an error on windows?

Reply:
On Windows, or Linux if extended file attributes aren't supported or doesn't exist, getXAttr will just error and fall through to the original file hashing step.  Do you think there'd be value in explicitly saying "not supported on Windows"?

Reply:
IIUC the code just returns "0, nil" on windows: https://github.com/pkg/xattr/blob/master/xattr_unsupported.go#L12, which would likely make the code fail somewhere down the pipeline. Having an explicit error would be easier to root-cause.

Reply:
You are correct, I was wrong.  I should've dug deeper.  Thank you, I'll add some checks.

Reply:
Check added